google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -192,3 +192,25 @@ def _auth_init(self):
 
         self._credentials = credentials
         self._cloud_sql_service = cloudsql
+
+    def _perform_refresh(self) -> Dict[str, Union[Dict, str]]:
+        """Retrieves instance metadata and ephemeral certificate from the
+        Cloud SQL Instance.
+
+        :rtype: Dict[str, Union[Dict, str]]
+        :returns: A Dictionary containing the server's certificate authority,
+            a Dictionary containing the IP addresses of the instance, and
+            the latest ephemeral certificate.
+        """
+        metadata_result = self._get_metadata(
+            self._cloud_sql_service, self._project, self._instance
```

Comment:
Does `self._get_metadata` need to be an object method? If you are already passing in the required attributes, would it be better as static instead?